bin/resources/GameIndex.yaml

```diff
@@ -30794,7 +30818,10 @@ SLPM-66613:
   name: "Medal of Honor - Rising Sun & Frontline [EA Best Hits]"
   region: "NTSC-J"
 SLPM-66615:
-  name: "007 - Everything or Nothing & Nightfire [EA Best Hits]"
+  name: "007 - Nightfire [EA Best Hits - 007: Nightfire & Everything or Nothing Dual Pack]"
```

Comment:
: is a disallowed character in filenames, please replace with a hyphen (-)

Reply:
actually I would say that how it was before (maybe mention double pack) was better? mentioning the whole game name twice seems redundant.